src/main/java/com/synopsys/integration/detect/lifecycle/run/DetectRun.java

```diff
@@ -80,6 +94,20 @@ public void run(BootSingletons bootSingletons) {
             } else {
                 bdio = BdioResult.none();
             }
+
+            if(autonomousManager.getAutonomousScanEnabled()) {
+                SortedMap<String, SortedSet<String>> packageManagerTargets = stepRunner.getScanTargets(universalToolsResult);
+                autonomousManager.updateScanTargets(packageManagerTargets, scanTypeEvidenceMap);
+                AllEnumList<DetectTool> givenScanTypes = bootSingletons.getDetectConfiguration().getValue(DetectProperties.DETECT_TOOLS);
+                List<String> scanTypes = new ArrayList<>();
+                scanTypeEvidenceMap.keySet().forEach(tool -> scanTypes.add(tool.toString()));
+                givenScanTypes.representedValues().forEach(tool -> scanTypes.add(tool.toString()));
+                packageManagerTargets.keySet().forEach(tool -> scanTypes.add(tool));
+                SortedMap<String, String> defaultValueMap = DetectProperties.getDefaultValues();
+                autonomousManager.updateScanSettingsProperties(defaultValueMap, scanTypes);
+                autonomousManager.writeScanSettingsModelToTarget(); //TODO - Move this call to Detect shutdown
```

Comment:
I have moved this call to OperationRunner in my branch and wrapped it as an Operation in Detect so that it shows up correctly in status.json and logs. Makes sense to keep it in DetectRun.